### PR TITLE
Add PSR-3 meta document to include type upgrades.

### PR DESCRIPTION
**Pull request by Crell**
See related code PRs:

v2: https://github.com/php-fig/log/pull/76
v3: https://github.com/php-fig/log/pull/77

### CODE REVIEW

File: accepted/PSR-3-logger-interface-meta.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Implementers MAY add return types to their own packages at their discretion, provided that:
+
+* the return types match those in the 3.0 package.
+* the implementation specifies a minimum PHP version of 8.0.0 or later.
+
+Implementers MAY add parameter types to their own packages in a new major release, either at the same time as adding return types or in a subsequent release, provided that:
+
+* the parameter types match those in the 2.0 package.
+* the implementation specifies a minimum PHP version of 8.0.0 or later.
+* the implementation depends on `"psr/link": "^2.0 || ^3.0"` so as to exclude the untyped 1.0 version.
```
Comment (shadowhand) [inline accepted/PSR-3-logger-interface-meta.md:42]: Is this the right package reference?

Comment (Crell) [reply accepted/PSR-3-logger-interface-meta.md:42]: Mm, yummy copy pasta.  Will fix.

**Review by shadowhand — APPROVED** [review]

File: accepted/PSR-3-logger-interface-meta.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+* Jordi Boggiano
+
+## 4. Votes
+
+[Approval vote](https://groups.google.com/g/php-fig/c/d0yPC7jWPAE/m/rhexAfz2T_8J)
+
+## 5. Errata
+
+### 7.1 Type additions
```
Comment (vdelau) [inline accepted/PSR-3-logger-interface-meta.md:29]: Should this be 5.1?

Comment (Crell) [reply accepted/PSR-3-logger-interface-meta.md:29]: Yes.  Yes it should.

**Review by vdelau — APPROVED** [review]

**Pull request merged**